hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpointWithLegacy.java

```diff
@@ -222,37 +227,48 @@ public class TestNSSummaryEndpointWithLegacy {
           + CONTAINER_TWO_REPLICA_COUNT * BLOCK_TWO_LENGTH
           + CONTAINER_THREE_REPLICA_COUNT * BLOCK_THREE_LENGTH;
 
-  private static final long MULTI_BLOCK_KEY_SIZE_WITH_REPLICA
-          = LOCATION_INFO_GROUP_ONE_SIZE;
 
   private static final long LOCATION_INFO_GROUP_TWO_SIZE
```

Comment:
Same as the above comment.

Reply:
yes, no longer referenced, could be removed!